doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,4 +1,4 @@
-= The Neo4j Graph Data Science Client Manual v{docs-version}
+= Neo4j Graph Data Science Client v{docs-version}
```

Comment:
Is this title change consistent with other Neo4j docs products? Ie, name the docs product identical to the product it is documenting, rather than naming it as a 'manual'.

Reply:
It's not consistent, no. Some manuals don't even have their title on the index page but "introduction" or something else completely. Now, another thing is that we are trying to remove the word "manual" from the docs and either have nothing at all or use documentation or guide.

Reply:
I do like having an actual title so that there is a natural "first page". Not partial to "manual", so I'm fine to lose it.

Reply:
This was called "User Guide" a long time back but we changed to "Manual" to be in line with "Operations Manual" and "Cypher Manual". They seem to still be called "manual" actually. Will they also change?

Reply:
Yes, eventually!

Reply:
I would suggest to make this a part of a larger change in the structure, involving the renaming of this page and the "flattening" of the other pages (which are now children of this page in the nav).